test/unit/model/include.test.js

```diff
@@ -207,7 +208,9 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           include: [{ where: { active: false }, model: this.Project.scope('that') }],
         });
 
-        expect(options.include[0]).to.have.property('where').which.deep.equals({ active: false, that: false });
+        expect(options.include[0]).to.have.property('where');
+        expect(Utils.getComplexKeys(options.include[0].where)).to.deep.equal([Op.and]);
```

Comment:
Aah I see you test it manually, then you can ignore the first part of my previous comment :)

Can you add `// TODO [chai>5]: simplify once '.deep.equal' includes support for symbols (https://github.com/chaijs/chai/issues/1054)`